Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ConceptCollectionWorkspaceController.java

```diff
@@ -81,7 +86,47 @@ public String listProjectConceptCollection(@PathVariable("workspaceid") String w
 		model.addAttribute("workspaceId", workspaceId);
 		return "auth/workbench/workspace/conceptcollections";
 	}
+
+	@AccessPolicies({ @ElementAccessPolicy(type = CheckedElementType.WORKSPACE,paramIndex = 1, userRole = {RoleNames.ROLE_WORKSPACE_COLLABORATOR_ADMIN} )})
+	@RequestMapping(value = "auth/workbench/workspace/{workspaceid}/conceptcollectionsJson", method = RequestMethod.GET, produces = "application/json")
+	@ResponseBody
+	public String listWorkspaceConceptCollection(@PathVariable("workspaceid") String workspaceId, Model model) throws QuadrigaStorageException, QuadrigaAccessException, QuadrigaException {

```

Comment:
this should be analog to the dictionaries (in rest controller, return responseentity, etc.)